### PR TITLE
Add Example Tests

### LINKED ISSUE #2: (Docs) Improve README
Users who did not visit the website, should be able to see the examples and features

### PR DESCRIPTION
**Pull request by umeshchavda05**
I've added the example tests along with a table of contents to the Readme file. Please review.
This closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,353 @@ Documentation is accessible under [`docs/`](docs/)
...
+env:
+	host: example.com
+tests:
+	example:
+		steps:
+			- name: GET request
+				url: https://{{env.host}}
+				method: GET
+				check:
+					status: /^20/
```
Comment (mishushakov) [inline README.md:149]: Could you please redo this part? It has some spacing issues

**Review by mishushakov — CHANGES_REQUESTED** [review]
Hey, thank you for the pull request! We need a small little change before we can merge. Thanks!

**Comment by umeshchavda05** [comment]
> Hey, thank you for the pull request! We need a small little change before we can merge. Thanks!

I've redone it. Please take a look.

**Review by mishushakov — APPROVED** [review]
Awesome! We're merging it. Thanks for your contribution
Feel free to check out other issues that need help 😉

**Comment by umeshchavda05** [comment]
thanks a lot!

**Comment by sebastianwis** [comment]
Thanks for helping out, @umeshchavda05 !
We are happy to have you as contributor! 
Feel free to accompany us on the further journey in any way! :)

**Pull request merged**